### PR TITLE
Substrate Upgrade

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: runtime/src/anchor/mod.rs [hunk 1]
```diff
@@ -204,7 +204,7 @@ decl_module! {
             };
 
             let anchor_data_encoded = anchor_data.encode();
-            runtime_io::set_child_storage(&child_storage_key, anchor_id.as_ref(), &anchor_data_encoded);
+            runtime_io::storage::child_set(&child_storage_key, anchor_id.as_ref(), &anchor_data_encoded);
```
Comment (mikiquantum) [inline runtime/src/anchor/mod.rs]: does this mean that the newer version has changed the file/package structure?

File: scripts/init.sh [hunk 2]
```diff
@@ -5,12 +5,8 @@ set -e
...
 if [ -z $CI_PROJECT_NAME ] ; then
-   #rustup update nightly
+   rustup update nightly
    rustup update stable
 fi
 
-rustup target add wasm32-unknown-unknown --toolchain $RUST_TOOLCHAIN
-
-# Install wasm-gc. It's useful for stripping slimming down wasm binaries.
-command -v wasm-gc || \
```
Comment (mikiquantum) [inline scripts/init.sh]: I noticed that when I first built the project I needed this wasm-gc , is there any reason why we are removing it?

File: src/service.rs [hunk 3]
```diff
@@ -89,46 +75,64 @@ macro_rules! new_full_start {
             )?;
 
             import_setup = Some((babe_block_import, grandpa_link, babe_link));
-
             Ok(import_queue)
+        // })?
```
Comment (mikiquantum) [inline src/service.rs]: Is there any reason why we comment this? Is it just to isolate the problem first?

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -8,39 +8,35 @@
...
 
-use babe::AuthorityId as BabeId;
-use client::{
-    block_builder::api::{self as block_builder_api, CheckInherentsResult, InherentData},
-    decl_runtime_apis, impl_runtime_apis, runtime_api as client_api,
-};
-use grandpa::fg_primitives;
-use grandpa::{AuthorityId as GrandpaId, AuthorityWeight as GrandpaWeight};
-use primitives::{crypto::key_types, OpaqueMetadata};
+use primitives::OpaqueMetadata;
```
Comment (mikiquantum) [inline runtime/src/lib.rs]: Is this our implementation of the node-template file updated to the latest released?

File: runtime/src/anchor/mod.rs (hunk 1 above)
Comment (philipstanislaus) [reply runtime/src/anchor/mod.rs]: Exactly 👍

File: runtime/src/lib.rs (hunk 4 above)
Comment (philipstanislaus) [reply runtime/src/lib.rs]: Not sure I understand your question, but yes, I did follow the changes in the node-template file. We are using a bunch more than the default template though.

File: src/service.rs (hunk 3 above)
Comment (philipstanislaus) [reply src/service.rs]: Exactly, I commented it out to fix compilation issues, has been re-added now.

File: scripts/init.sh (hunk 2 above)
Comment (philipstanislaus) [reply scripts/init.sh]: True, the substrate team changed that requirement, the wasm optimization is happening within the wasm build stage now.

**Pull request merged**